lightspark/objects/PaymentRequestData.py

```diff
@@ -284,8 +284,8 @@ def from_json(requester: Requester, obj: Mapping[str, Any]) -> PaymentRequestDat
             ),
             payment_hash=obj["invoice_data_payment_hash"],
             amount=CurrencyAmount_from_json(requester, obj["invoice_data_amount"]),
-            created_at=obj["invoice_data_created_at"],
-            expires_at=obj["invoice_data_expires_at"],
+            created_at=datetime.fromisoformat(obj["invoice_data_created_at"]),
```

Comment:
Looks like we may need to import datetime too?

Reply:
Duh! Of course!